### ISSUE TITLE
Venom not start the client to send or receive messages

### ISSUE THREAD

**Issue by etc-tiago** [orkestral/venom#2020]
## Description

The venom not start client in the latest version. I tested in macOS and Ubuntu, with the firefox and chrome, but unable to test in chromium.

## Environment

- **Venom version(s):** 4.3.5
- **Browser:** Firefox 107, Chrome 107
- **OS:**  OSX 12.14, Ubuntu 20
- **Node version:** Node 16, Node 18

## Steps to Reproduce
1. Code
```ts
import { create } from "venom-bot";

const start = (client) => {
    console.log('client started');
}

create({
  session: 'dev',
  useChrome: false,
})
  .then((client) => {
    start(client);
  })
  .catch((error) => {
    console.error(error);
  });
```
1.1 The 'client started' not print on terminal, WhatsApp load on chrome normally, and the terminal stop with:
```sh
info:     [dev] Checking page...
info:     [dev] Page successfully accessed
✓ [instance: dev]: Connected
```

2. Code
```ts
import { create } from "venom-bot";

const start = (client) => {
    console.log('client started');
}

create({
  session: 'dev',
  useChrome: false, // <----- here
})
  .then((client) => {
    start(client);
  })
  .catch((error) => {
    console.error(error);
  });
```
2.1 The console
```
info:     https://orkestral.io - official site!

info:     [dev] Waiting... checking the browser...
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
```

---
*Update 1*

Using headless as false, I notice that an error ocurr:
```
VM132:4 Uncaught (in promise) ReferenceError: mergedOptions is not defined
    at eval (eval at createFunction (web.whatsapp.com/:466:8), <anonymous>:4:13)
    at IntervalPoller.<anonymous> (VM131 __puppeteer_evaluation_script__:4:36)
    at IntervalPoller.start (VM5 __puppeteer_evaluation_script__:198:34)
    at VM133 __puppeteer_evaluation_script__:2:24
```

---
*Update 2*

All works well in venom@4.3.3

**Comment by orkestral** [comment]
after some tests no problem found, please don't open this problem again

**Comment by orkestral** [comment]
resolved [8d89f6d](https://github.com/orkestral/venom/commit/8d89f6dbb0148594d2101ceb7db3e7e07f293c93)
